Hudl.Mjolnir/Command/CommandContext.cs

```diff
@@ -132,18 +143,18 @@ internal static ICircuitBreaker GetCircuitBreaker(GroupKey key)
                     new ConfigurableValue<bool>("mjolnir.breaker." + key + ".forceTripped", DefaultBreakerForceTripped),
                     new ConfigurableValue<bool>("mjolnir.breaker." + key + ".forceFixed", DefaultBreakerForceFixed));
 
-                return new FailurePercentageCircuitBreaker(key, metrics, Stats, properties);
+                return new FailurePercentageCircuitBreaker(key, metrics, Stats, MetricEvents, properties);
```

Comment:
I'll go through here and see if it feels alright to change the `LazyThreadSafetyMode` on these to lock. I'm a little wary of making the change (I'm not sure how much contention we have on these today), so I might do some measuring first if I can.